README.md

```diff
@@ -29,7 +29,7 @@
 ---
 
 [Website](https://druid.apache.org/) |
-[Documentation](https://druid.apache.org/docs/latest/design/) |
+[Documentation](#documentation) |
```

Comment:
It looks like the intention of this link is to point to the documentation like the others in this section. Suggest leaving as-is.